<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetFitterAutoselect</class>
 <widget class="QWidget" name="WidgetFitterAutoselect">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>615</width>
    <height>25</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources/qpxres.qrc">
    <normaloff>:/icons/qpx.xpm</normaloff>:/icons/qpx.xpm</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="label_51">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>25</height>
      </size>
     </property>
     <property name="text">
      <string>Autoselect new</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="comboAutoMethod">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>25</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QCheckBox" name="checkReselect">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>25</height>
      </size>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>Reselect on update within FWHM*</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QDoubleSpinBox" name="doubleSlack">
     <property name="minimumSize">
      <size>
       <width>70</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>70</width>
       <height>25</height>
      </size>
     </property>
     <property name="decimals">
      <number>2</number>
     </property>
     <property name="minimum">
      <double>0.000000000000000</double>
     </property>
     <property name="maximum">
      <double>10.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
     <property name="value">
      <double>0.500000000000000</double>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources/qpxres.qrc"/>
 </resources>
 <connections/>
</ui>
